docs/deployment/aws-secureboot.md

```diff
@@ -33,20 +36,80 @@ make aws-secureboot-dev BUILD_OPTS="--lessram"
 
-Once you have build a new Garden Linux image using the secure boot feature, there will also be signature databases which are required for secure boot. These databases must be uploaded to AWS as UEFI vars, so that UEFI can verify the integrity of the booted system during the startup process.
+Besides of this, you can also enable secure boot externally. This way, you have secure boot enabled right at the first start, but you need to provide all information during  the creation of the instances, which also requires some additional tools to be executed on your workstation.
+
+### Option A: Enable secure boot internally
+
+By creating a Garden Linux with the secure boot feature enabled and by using UEFI's setup mode, you can activate secure boot from within the AWS instance once the Garden Linux image has been spawned successfully. As long as you do not enable secure boot from within the instance, the Garden Linux image will simply spawn without secure boot enabled. This must be kept in mind.
+
+#### Integration test configuration
+Before starting the required AWS environment, a configuration for the integration test platform is needed.
+
+The following example shows such a configuration. For test purposes, it uses the `eu-west-1` region, a small instance type (e.g. `t3.micro`), an image with `amd64` architecture and most importantly the `UEFI` boot mode.
+
+Generally, there is no need to adjust this configuration, if you have created the Garden Linux image as described in this [chapter](#create-image).
+
+However, you may need to adjust the path to your image artifact which could look like `aws-gardener_secureboot_dev-amd64-today-<commit hash|local>.raw`.
+
+The configuration looks like this then:
+```
+aws:
+    # region
+    region: eu-west-1
+    # machine/instance type of the test VM - not all machines are available in all regions (optional)
+    instance_type: t3.micro
+    # architecture of the image and VM to be used (optional)
+    architecture: amd64
+    # boot mode
+    boot_mode: uefi
+    # UEFI data. This contains the UEFI variables
+    uefi_data: <content of secure_boot_blob.bin>
+
+    # local/S3 image file to be uploaded for the test (optional/alternatively required)
+    image: file:/build/aws-gardener_secureboot_dev-amd64-today-local.raw
+
+    # keep instance running after tests finishes (optional)
+    keep_running: true
+
+```
+
+Finally, the content must be saved into a YAML file and it has to be mounted to the integration test container later on. For this case, create a folder in your home directory and place the YAML in there. This folder will be mounted to the integration test container afterwards. The configuration could be called `aws-secureboot.yaml` for example.
```

Comment:
`For this case, create a folder in your home directory and place the YAML in there.`

In my home dir? Hopefully it couldn't access this one without any further todo. Probably the project's root is meant?